app/controllers/ui_job_wizard_controller.rb

```diff
@@ -24,6 +24,19 @@ def map_template_inputs(template_inputs_with_foreign)
     template_inputs_with_foreign.map { |input| input.attributes.merge({:resource_type_tableize => input.resource_type&.tableize }) }
   end
 
+  def default_category
+    default_template&.job_category
+  end
+
+  def default_template
```

Comment:
This is almost the same as https://github.com/theforeman/foreman_remote_execution/blob/master/app/models/job_invocation_composer.rb#L643 . For now I wouldn't mind having it duplicated and I'd like to see it eventually removed from the composer once we fully move to the new form.

However, the implementation in composer currently checks permissions, which this one does not. We should check them here as well